<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth" >
      <number>4</number>
     </property>
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="TextView" />
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="CommandEdit" name="TextEntry" />
         </item>
         <item>
          <widget class="QPushButton" name="SendButton" >
           <property name="text" >
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QListView" name="NickView" />
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommandEdit</class>
   <extends>QLineEdit</extends>
   <header>commandedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>TextEntry</sender>
   <signal>returnPressed()</signal>
   <receiver>SendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>67</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel" >
     <x>147</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
